src/NHibernate.Test/Linq/MathFTests.cs

```diff
@@ -7,7 +7,6 @@
 
 namespace NHibernate.Test.Linq
 {
-	[TestFixture]
 	public class MathFTests : LinqTestCase
```

Comment:
I guess this class was not having async test generated due to being netcoreapp only, when we were generating for netFx.

Have you removed the attribute to avoid having its asyn tests generated? Why not having them?

Reply:
> I guess this class was not having async test generated due to being netcoreapp only, when we were generating for netFx.

Correct.

> Have you removed the attribute to avoid having its asyn tests generated?

Yes, otherwise the generated tests would fail as `ToListAsync` would be called on a queryable `List<>` which is populated on set up:
https://github.com/nhibernate/nhibernate-core/blob/8f765cf6b18fe5fb5eb934387677c97a3370dee0/src/NHibernate.Test/Linq/MathFTests.cs#L17-L19

Reply:
I had to test it to understand. So the trouble is, the async generator has no way to understand on which queryable using the NHibernate `ToListAsync` is valid or not.
This means the async generator cannot be used in an application which would use NHibernate and some other queryables.
That is off-topic for this PR, but maybe we should then relax the `ToListAsync` implementation for defaulting to `ToList` on not handled queryables, with some warning, or with an option to enable not throwing.